### PR TITLE
implementation of stack

### PR DESCRIPTION
**Pull request by Rasingan**
Signed-off-by: Rasingan <siddheshrainbow@gmail.com>

### CODE REVIEW

**Comment by Rasingan** [comment]
@ksjc1995 
kindly go through it , will be adding html codes asap.

File: Python/Stack/implementation.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    # As soon as class stack is called , init Function works and create an empty Stack
+    def __init__(self):
+        self.stack = []
+
+    # Push operation , we insert element in stack 1 by 1
+    def push(self,element):
+        self.stack.append(element)
+
+    # we remove element from stack
+    def pop(self):
```
Comment (ksjc1995) [inline Python/Stack/implementation.py:31]: I think we need a check for underflow condition (i.e) when stack is empty we can't pop.

Comment (ksjc1995) [reply Python/Stack/implementation.py:31]: `if(self.is_empty) return "Underflow";`

File: Python/Stack/implementation.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        self.stack.pop()
+
+    # we don't pop the top element , just view it 
+    def top(self):
+        if self.is_empty:
```
Comment (ksjc1995) [inline Python/Stack/implementation.py]: Shouldn't this be `self.is_empty()` ? as `is_empty` is a function right?

File: Python/Stack/implementation.py (hunk 1 above)
Comment (ksjc1995) [reply Python/Stack/implementation.py:31]: & also i think we should return the popped element

**Review by ksjc1995 — COMMENTED** [review]
Please add test cases as well for a variety of inputs as discussed

**Comment by Rasingan** [comment]
@ksjc1995 
seperated test cases as discussed.

**Review by ksjc1995 — APPROVED** [review]

**Pull request merged**